packages/webpack/lib/plugins/log.js

```diff
@@ -70,6 +70,15 @@ exports.LoggerPlugin = class LoggerPlugin {
           this.logger.info(formatWarning(name, duration, assets, isRebuild));
         } else {
           this.logger.info(formatSuccess(name, duration, assets, isRebuild));
+          const assetsOverSizeLimit = assets.some(
+            ({ isOverSizeLimit }) => isOverSizeLimit
+          );
+          if (assetsOverSizeLimit) {
+            this.logger.warn('Some assets exceed the recommended size limit!');
+            this.logger.hint(
+              'You can limit the bundle size by using the importComponent() helper to lazy load some parts of the application.'
```

Comment:
I am not too keen on this hint as it is as there are plenty of other options to limit bundle sizes (`import()`/`require.ensure()`, dependency deduplication or elimination, etc).

This wording makes it sound like `importComponent` was the only way to limit bundle size. If we are convinced we need a hint like this, how about `Please consider using import() or the importComponent() helper to lazy load some parts of the application`?

Reply:
I changed to wording to your suggestion for now. Maybe we can find a way to have a readme section / wiki article on more in-depth knowledge for analysing and reducing the bundle sizes later on.